src/app/pagamentoplanos/service/external.service.ts

```diff
@@ -0,0 +1,57 @@
+import {Injectable} from "@angular/core";
+import {ScriptStore} from "./script.store";
+
+declare var document: any;
+
+@Injectable()
+export class ExternalService {
+
+private scripts: any = {};
+
+constructor() {
+    ScriptStore.forEach((script: any) => {
+        this.scripts[script.name] = {
+            loaded: false,
+            src: script.src
+        };
+    });
+}
+
+load(...scripts: string[]) {
+    var promises: any[] = [];
+    scripts.forEach((script) => promises.push(this.loadScript(script)));
+    return Promise.all(promises);
+}
+
+loadScript(name: string) {
+    return new Promise((resolve, reject) => {
+        //resolve if already loaded
+        if (this.scripts[name].loaded) {
+            resolve({script: name, loaded: true, status: 'Already Loaded'});
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PadawansTeam/artmais-frontend/pullRequest?prid=8398929)
```suggestion
            resolve({script: name, loaded: true, status: "Already Loaded"});
```